tox.ini

```diff
@@ -119,6 +119,7 @@ changedir = testing/freeze
 # Disable PEP 517 with pip, which does not work with PyInstaller currently.
 deps =
     pyinstaller
+    setuptools < 45.0.0
```

Comment:
Needed to make the installation of `pyinstaller` successful. Check out https://github.com/pypa/setuptools/issues/1963